definitions/npm/commander_v2.x.x/test_commander_v2.x.x.js

```diff
@@ -0,0 +1,13 @@
+import program from 'commander';
```

Comment:
Let's add some [`$ExpectError`s](https://github.com/flowtype/flow-typed/blob/master/CONTRIBUTING.md#4-write-a-test-file-whose-name-starts-with-test_-in-the-package-version-directory) in these tests too, pls